client/src/components/DataImportExport.vue

```diff
@@ -40,7 +40,9 @@ export default defineComponent({
           timeout: 6000,
         });
 
-        await loadProject(currentProject.value);
+        if (currentProject.value.id !== 'global') {
```

Comment:
Would it be difficult to have global indicated by `currentProject === null` rather than this special `id` value?

Reply:
changed in f932555